### ISSUE TITLE
Smart Select doesn't show me anything

### ISSUE THREAD

**Issue by SalahAdDin** [digi604/django-smart-selects#114]
Hi guys? I have the next problem, i'm update django-smart-selects and now it doesn't works.

I don't know why, at first, i think that problem was django-jet:
![mozilla firefox_047](https://cloud.githubusercontent.com/assets/5034215/12772251/a380fb90-c9fd-11e5-8a2b-20b616666929.png)
I have a chained foreingkey between affiliates withg membership through entity's name. I select a Entity and Membership field doesn't load anything.
![mozilla firefox_050](https://cloud.githubusercontent.com/assets/5034215/12772252/a39948c6-c9fd-11e5-9fcc-62830794c3a3.png)
First, if i don't put `USE_DJANGO_JQUERY = False` i haven't any jquery load file, yes, i haven't now jquery and i think i won't put jquery in admin/js, will be great put a custom jquery file path.
![mozilla firefox_051](https://cloud.githubusercontent.com/assets/5034215/12772254/a3b23304-c9fd-11e5-8edb-0c0881ee66a8.png)
Ok, i put `USE_DJANGO_JQUERY = False` and this load a Google CDN jquery, but, wrong version. I have these problem and i don't understand which warning is.
![mozilla firefox_052](https://cloud.githubusercontent.com/assets/5034215/12772253/a3b04cf6-c9fd-11e5-9dde-67d534a1a63e.png)
Ok, i put the newest jquery  Google CDN link and nothing errors but, can you see any estrange here? Oh yeah, anything, memberships fields load nothing. Why?
![mozilla firefox_053](https://cloud.githubusercontent.com/assets/5034215/12772255/a3c93d1a-c9fd-11e5-8402-b45d0fc51825.png)
You can think, there aren't any membership with juridical entity type, isn't, as you can see.

You can think, it's a problem from django-jet, good, i though this and i test without django-jet:
![add affiliate django site admin - mozilla firefox_048](https://cloud.githubusercontent.com/assets/5034215/12772506/b5c53f3a-c9ff-11e5-8c15-9c311adbc6da.png)
 I test each one of the previous steps and i have the same problems.

So, i don't know why happen this if with previous django-smart-selects versions i hadn't problems. And, i haven't idea for what do, because, as you can see, i haven't any errors in firefox console, so, i'm lost.

```
System check identified no issues (0 silenced).
February 02, 2016 - 22:56:01
Django version 1.9.2, using settings 'ccct.settings'
Starting development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.
```

Django info.

Thank you for your help

PD: Where are Form's build docs? I have do anything for use smart_selects in forms?

Anyone have problem with the actual version?

@jimmykobe1171 What's about this?

**Comment by jimmykobe1171** [comment]
@SalahAdDin could you have a try of the version on branch master? I have fix some Jquery issue on Django 1.9.

**Comment by blag** [comment]
@SalahAdDin Please test against master.

**Comment by blag** [comment]
Version 1.2.6 - which was released on PyPI a few days ago - fixed a few issues with no results being returned and a previous released version allowed for disabling automatically loading jQuery (use `USE_DJANGO_JQUERY = None` and load jQuery yourself for that - see the updated README for more).

I'm going to close this because I suspect that both issues are now fixed, but feel free to reopen if they aren't fixed in version 1.2.6 or the `master` branch on GitHub. Thanks!

**Comment by SalahAdDin** [comment]
Ok, thank you!
